CHANGELOG.md

```diff
@@ -7,6 +7,11 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
+## [0.0.37] - 2021-12-08
```

Comment:
missing 0.0.38 ?

Reply:
My workflow is mainly to let changelog and bump version when all is ready ^^